util/mkdef.pl

```diff
@@ -108,6 +108,7 @@
                      platforms  => { UNIX                       => 1,
                                      EXPORT_VAR_AS_FUNCTION     => 0 } },
     linux       => 'solaris',   # alias
+    "bsd-gcc"   => 'solaris',   # alias
```

Comment:
vim's syntax highlighting complained about an unquoted ```bsd-gcc``` implying that the ```-``` would be treated as an operator.  I don't think I actually tried running things with the unquoted version, though.